### ISSUE TITLE
Don't add compartment postfix to metabolite identifiers

### ISSUE THREAD

**Issue by kvikshaug** [DD-DeCaF/caffeine-vue#95]
Currently, we [add a compartment postfix to any metabolite id from added reactions](https://github.com/DD-DeCaF/caffeine-vue/blob/2cc45146281b8268cf401f9dcbcbbac4cf858c73/src/views/InteractiveMap/InteractiveMap.vue#L373). That was relevant when adding reactions from BiGG, but now that we're adding any reaction from metanetx, this logic should be removed. That would allow us to remove for example [this](https://github.com/DD-DeCaF/caffeine-vue/blob/2cc45146281b8268cf401f9dcbcbbac4cf858c73/src/views/InteractiveMap/ReactionDialog.vue#L271) and [this](https://github.com/DD-DeCaF/caffeine-vue/blob/2cc45146281b8268cf401f9dcbcbbac4cf858c73/src/views/InteractiveMap/CardDialogDesign.vue#L397) piece of logic.

**Comment by Midnighter** [comment]
You mean any non-native metabolite? Because metabolites from added reactions might be mapped to native metabolites that are then in BiGG notation and need to have the compartment suffix?

**Comment by kvikshaug** [comment]
Yes, indeed. We'd still use the postfix when looking for native metabolites (it would be nice to confirm that the model is in BiGG namespace before doing that though).

In effect, the changes I'm thinking of with this issue is:

- When adding a native metabolite, do include the postfix in the metabolite identifier ([currently removed here](https://github.com/DD-DeCaF/caffeine-vue/blob/2cc45146281b8268cf401f9dcbcbbac4cf858c73/src/views/InteractiveMap/CardDialogDesign.vue#L397))
- Don't remove the compartment postfix [here](https://github.com/DD-DeCaF/caffeine-vue/blob/2cc45146281b8268cf401f9dcbcbbac4cf858c73/src/views/InteractiveMap/ReactionDialog.vue#L325)
- When simulating, use the identifier directly [without adding the postfix here](https://github.com/DD-DeCaF/caffeine-vue/blob/2cc45146281b8268cf401f9dcbcbbac4cf858c73/src/views/InteractiveMap/InteractiveMap.vue#L373)
